src/bun.js/bindings/ZigGlobalObject.cpp

```diff
@@ -3323,6 +3323,13 @@ void GlobalObject::installAPIGlobals(JSClassRef* globals, int count, JSC::VM& vm
                 JSC::PropertyAttribute::Function | JSC::PropertyAttribute::DontDelete | 0);
             object->putDirect(vm, PropertyName(identifier), dnsObject, JSC::PropertyAttribute::ReadOnly | JSC::PropertyAttribute::DontDelete | 0);
         }
+        {
+            JSC::Identifier identifier = JSC::Identifier::fromString(vm, "dns"_s);
+            JSC::JSObject* dnsObject = JSC::constructEmptyObject(this);
+            dnsObject->putDirectNativeFunction(vm, this, JSC::Identifier::fromString(vm, "resolveSrv"_s), 2, Bun__DNSResolver__resolveSrv, ImplementationVisibility::Public, NoIntrinsic,
+                JSC::PropertyAttribute::Function | JSC::PropertyAttribute::DontDelete | 0);
+            object->putDirect(vm, PropertyName(identifier), dnsObject, JSC::PropertyAttribute::ReadOnly | JSC::PropertyAttribute::DontDelete | 0);
```

Comment:
This overrides the existing `dns` object, removing the one with a `lookup` function defined

Instead, add the `resolveSrv` property onto the `dnsObject` above

Reply:
done